src/test/unit/process-profile.test.js

```diff
@@ -237,21 +235,26 @@ describe('gecko counters processing', function() {
     const offsetTime = originalTime.map(n => n + 1000);
 
     const extractTime = counter => {
-      if (!counter.sample_groups.samples) {
+      if (
+        counter.sample_groups.length !== 1 ||
+        !counter.sample_groups[0].samples
```

Comment:
I don't think this second part of the condition is possible with your new code, right?
Can you please simplify this?

Also how about using `=== 0` instead of ` !==1`?